### PR TITLE
Create LIT7066NiceneCreedAmh.xml

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
Created file based on https://github.com/BetaMasaheft/Documentation/issues/2624#issuecomment-2328937856

### CODE REVIEW

File: new/LIT7066NiceneCreedAmh.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT7066NiceneCreedAmh.xml:24]: Add witness UppEt10 to source desc.

File: new/LIT7066NiceneCreedAmh.xml
Comment (nafisa-valieva) [inline new/LIT7066NiceneCreedAmh.xml:1]: (Amharic) in title looks a bit strange, if I may have my opinion. We do not write (Geez)  :-)

Comment (CarstenHoffmannMarburg) [reply new/LIT7066NiceneCreedAmh.xml:1]: It is not consistent, but for practical reasons very useful!

File: new/LIT7066NiceneCreedAmh.xml (hunk 1 above)
Comment (karljonaskarlsson) [reply new/LIT7066NiceneCreedAmh.xml:24]: I thought we used <listWit> for listing the manuscripts used in an edition?

Comment (karljonaskarlsson) [reply new/LIT7066NiceneCreedAmh.xml:24]: Ah, I understand that the witness used for our edition (= incipit, in this case) can be listed as a witness, so I will add UppEt10, as you suggested. Thank you!

**Review by eu-genia — APPROVED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Pull request merged**